inst/examples/RcppProgressETA/src/eta_progress_bar.hpp

```diff
@@ -19,141 +19,168 @@
 
 // for unices only
 #if !defined(WIN32) && !defined(__WIN32) && !defined(__WIN32__)
-#include <Rinterface.h>
+  #include <Rinterface.h>
```

Comment:
typo ?

Reply:
hm? What's wrong here? I just like indentation... Is this against a common style guide?

Reply:
I think so. preprocessor stuff should be in the first column.

Reply:
some old compilers require that, moreover:
> Usually they're put at the beginning of the line because they're not subjected to the scopes > they're into, since they're preprocessed before actual C code.

anyway, since the other source files follow this (implicit) convention, better to stick to it even if no rationale...

Reply:
Thanks for the explanation - I removed tabs before preprocessor directives.